src/s3/putobject.cpp

```diff
@@ -110,4 +119,4 @@ asio::awaitable<void> irods::s3::actions::handle_putobject(
     }
     co_return;
 }
-#pragma clang diagnostic pop
\ No newline at end of file
+#pragma clang diagnostic pop
```

Comment:
Let's plan to get rid of the `#pragma`s. Please capture in an issue.

Reply:
done